lib/core/src/chain/liquid.rs

```diff
@@ -26,7 +27,7 @@ const LIQUID_ESPLORA_URL: &str = "https://lq1.breez.technology/liquid/api";
 #[async_trait]
 pub trait LiquidChainService: Send + Sync {
     /// Get the blockchain latest block
-    async fn tip(&mut self) -> Result<u32>;
+    async fn tip(&mut self) -> Result<BlockHeader>;
```

Comment:
Do we still want to keep this change now? It was only made to expose the BlockHeader time and we no longer use it